bundles/org.openhab.binding.fronius/README.md

```diff
@@ -1,54 +1,89 @@
+| `inverterdatachanneliac`             | Number    | AC current                                                                                                        |
+| `inverterdatachannelidc`             | Number    | DC current                                                                                                        |
+| `inverterdatachanneluac`             | Number    | AC voltage                                                                                                        |
+| `inverterdatachanneludc`             | Number    | DC voltage                                                                                                        |
+| `inverterdatadevicestatuserrorcode`  | Number    | Device error code                                                                                                 |
+| `inverterdatadevicestatusstatuscode` | Number    | Device status code<br />`0` - `6` Startup<br />`7` Running <br />`8` Standby<br />`9` Bootloading<br />`10` Error |
+| `powerflowchannelpgrid`              | Number    | Power + from grid, - to grid                                                                                      |
+| `powerflowchannelpload`              | Number    | Power + generator, - consumer                                                                                     |
+| `powerflowchannelpakku`              | Number    | Power + charge, - discharge                                                                                       |
+
+### Channels for `meter` Thing
+
+| Channel ID              | Item Type | Description                                                                                                                                                                                                              |
+| ----------------------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `model`                 | String    | The model name of the meter                                                                                                                                                                                              |
+| `serial`                | String    | The serial number of the meter                                                                                                                                                                                           |
+| `enable`                | Number    | 1 = enabled, 0 = disabled                                                                                                                                                                                                |
```

Comment:
What's the purpose of this Channel?

Reply:
I presume you are referring to `enable`? It is returned by the json data from the smartmeter to indicate whether this particular smartmeter is enabled or disabled. The name `enable` is just mimicking the naming from the json data. IMO it should be called `enabled` but I don't want to confuse people who refer to the api document, so I decided to keep the naming the same.